src/main/scala/cromwell/backend/provider/local/LocalBackendActor.scala

```diff
@@ -88,9 +89,9 @@ class LocalBackendActor(task: TaskDescriptor) extends BackendActor with StrictLo
       val command = initiateCommand()
       logger.debug(s"Creating bash script for executing command: $command.")
       writeBashScript(command, executionDir)
-      notifyToSubscribers(new TaskStatus(Status.Created))
+      notifySubscribers(CreatedTaskStatus(stdout.toString, stderr.toString))
```

Comment:
Yes, that part I understand, same happen with AWS S3, but what I don't understand what you do with that in the engine... does it know how to access to GCP? or it's being delegated to another component in the engine?